usr/share/rear/layout/save/GNU/Linux/260_crypt_layout.sh

```diff
@@ -95,9 +136,9 @@ while read target_name junk ; do
     # and it seems cryptsetup fails when options with empty values are specified
     # cf. https://github.com/rear/rear/pull/2504#issuecomment-719479724
     # For example a LUKS1 crypt entry in disklayout.conf looks like
-    # crypt /dev/mapper/luks1test /dev/sda7 type=luks1 cipher=aes-xts-plain64 key_size=256 hash=sha256 uuid=1b4198c9-d9b0-4c57-b9a3-3433e391e706 
+    # crypt /dev/mapper/luks1test /dev/sda7 type=luks1 cipher=aes-xts-plain64 key_size=256 hash=sha256 uuid=1b4198c9-d9b0-4c57-b9a3-3433e391e706
     # and a LUKS1 crypt entry in disklayout.conf looks like
-    # crypt /dev/mapper/luks2test /dev/sda8 type=luks2 cipher=aes-xts-plain64 key_size=256 hash=sha256 uuid=3e874a28-7415-4f8c-9757-b3f28a96c4d2 
+    # crypt /dev/mapper/luks2test /dev/sda8 type=luks2 cipher=aes-xts-plain64 key_size=256 hash=sha256 uuid=3e874a28-7415-4f8c-9757-b3f28a96c4d2 [pbkdf=argon2id]
```

Comment:
is `[pbkdf=argon2id]` really within brackets in disklayout.conf
or are the brackets used here only to denote that `[pbkdf=...]` is optional?
In the latter case please remove the brackets and show how it
literally looks in disklayout.conf because other options
which are shown here in this example 'crypt' entry are also
only optional.

Additionally please update in
doc/user-guide/06-layout-configuration.adoc
in the "Disk layout file syntax" section
the part about "LUKS Devices" i.e.
the 'crypt' syntax part.

Reply:
yeah just optional, removing this.